### ISSUE TITLE
ScreenToGif.2.9.Portable.zip is not portable

### ISSUE THREAD

**Issue by AdKiller** [NickeManarin/ScreenToGif#176]
it creates settings in AppData, uses system TEMP folder and maybe registry data too...

**Comment by NickeManarin** [comment]
It's portable but non-stealth, refer to this link:

https://www.portablefreeware.com/?id=2895&ts=1499027639

**Comment by AdKiller** [comment]
I discovered the dots for TEMP folder location and moving the Settings.xaml. but IMHO they should be already included with portable zip, or at least described next to the download link.

**Comment by vatterspun** [comment]
@AdKiller : I think NickeManarin is going for a "noinstall" version that a lot of other programs produce.  This is great if installer access has been disabled by the admin, but of course comes up short when you want to use it on a USB or over a cloud service.

You can alternatively go into **Options - Temporary Files** and click the **+** symbol next to **Local**

@NickeManarin : PFW will update the instructions if you decide to add the _Settings.xaml_ file by default in the distro.

**Comment by NickeManarin** [comment]
I changed the description of the download to "no install" instead of "portable", but yes an instruction would be nice. :)

@vatterspun I was thinking about that but I decided that I won't bundle the _Settings.xaml_ file with the zip. I think that it's easier for a tech-savvy person to deal with this situation (provided that a description is available). 

I'll create a wiki page and link it on the page later.
